python-checks/src/main/java/org/sonar/python/checks/ModuleNameCheck.java

```diff
@@ -49,20 +48,22 @@
   private Pattern pattern = null;
 
   @Override
-  public void init() {
-    pattern = Pattern.compile(format);
-  }
-
-  @Override
   public void visitFile(@Nullable AstNode astNode) {
-    String fileName = getContext().getFile().getName();
+    String fileName = getContext().pythonFile().file().getName();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this use of "file"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)